pkg/tenant/api/v1/client.go

```diff
@@ -266,12 +266,12 @@ func (s *APIv1) TenantDelete(ctx context.Context, id string) (err error) {
 	return nil
 }
 
-func (s *APIv1) TenantStats(ctx context.Context, id string) (out []*StatCount, err error) {
+func (s *APIv1) TenantStats(ctx context.Context, id string) (out []*StatValue, err error) {
 	if id == "" {
 		return nil, ErrTenantIDRequired
 	}
 
-	path := fmt.Sprintf("/v1/tenant/%s/stats", id)
+	path := fmt.Sprintf("/v1/stats/tenant/%s", id)
```

Comment:
I know that you and Cletus discussed this, but I didn't see in the story why swapping the root of the route would reduce contention. Would you mind giving me a quick run down of the discussion?

Reply:
I don't know the exact issue but the frontend was having trouble distinguishing the `/v1/tenant/:tenantID/projects` and  `/v1/tenant/:tenantID/stats` patterns. My suggestion was to move the stats endpoints to a different hierarchy since we might have more of them in the future.

Reply:
hmm, ok - but likely we won't be willing to do that for other things in the object hierarchy so they will have to manage those routes at some point. Personally, I think the prefixes are easier because you can store the prefix and build all follow-on routes off them easily. With the reversal for `stats` that's no longer true.